syntax = "proto3";

package payment;

service PaymentService {
    rpc PreparePayment(PrepareRequest) returns (PrepareResponse);
    rpc CommitPayment(CommitRequest) returns (CommitResponse);
}

message PrepareRequest {
    string orderId = 1;
}

message PrepareResponse {
    bool isReady = 1;
}

message CommitRequest {
    string orderId = 1;
}

message CommitResponse {
    bool isSuccess = 1;
}